bus/src/main/java/org/hawkular/metrics/component/publish/PublishCommandMessage.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2014-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.metrics.component.publish;
+
+import java.util.List;
+
+import org.hawkular.bus.common.AbstractMessage;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * A bus message that will indicate which metrics will be published/unpublished to the bus.
```

Comment:
Let's perhaps not call this published to the bus but rather just published for Alerting consumption.  I think what we're doing here is more tightly coupling these two components, which is not a bad thing.